test/e2e/scaffold/k8s.go

```diff
@@ -29,11 +28,12 @@ import (
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 type counter struct {
-	Count string `json:"count"`
+	Count intstr.IntOrString `json:"count"`
```

Comment:
Are you sure the marshaling and unmarshaling can be done?

type IntOrString struct {
	Type   Type   `protobuf:"varint,1,opt,name=type,casttype=Type"`
	IntVal int32  `protobuf:"varint,2,opt,name=intVal"`
	StrVal string `protobuf:"bytes,3,opt,name=strVal"`
}

Reply:
This can not be work. 
Done. Customize the `intOrString` type.